### PR TITLE
docs($zh): add a reference link to styling config

### PR DESCRIPTION
**Pull request by ChungZH**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/zh/config/README.md [hunk 1]
```diff
@@ -179,6 +179,10 @@ VuePress 提供了一种添加额外样式的简便方法。你可以创建一
 }
 ```
 
+**还可以看看：**
```
Comment (newsbielt703) [inline packages/docs/docs/zh/config/README.md:182]:
```suggestion
**参考:**
```

**Comment by newsbielt703** [comment]
Thanks. Could you also update it in English doc?

**Comment by ChungZH** [comment]
Actually this link is already in English version, I just moved over

**Comment by newsbielt703** [comment]
Oops, I see.

**Pull request merged**